py-polars/polars/utils/udfs.py

```diff
@@ -192,6 +192,14 @@ class OpNames:
         "function_name": [{"strptime"}],
     },
 ]
+_UNARY_OPNAME_POSSIBILITIES: list[list[AbstractSet[str]]] = [[set(OpNames.UNARY)], []]
```

Comment:
Minor nitpick; this doesn't actually seem to expand the unary operator possibilities, it just allows the existing unary ops to be optionally present when matching? In which case I'd just bake it in directly like so...
```python
_FUNCTION_KINDS = [
    {**kind, "argument_1_unary_opname": unary}
    for kind in _FUNCTION_KINDS
    for unary in [[set(OpNames.UNARY)], []]
]
```

Reply:
thanks! i only did this to appease mypy, maybe a `type ignore` is warranted here

Reply:
Works for me :)